python.md

```diff
@@ -38,6 +38,7 @@ General Python:
 
 - [Wrap with parens not backslashes](#wrapping)
 - [Import modules, not objects](#import-modules-not-objects)
+- [Avoid wildcard imports](#avoid-wildcard-imports)
```

Comment:
This link is broken - I'm not 100% what it should be though

Reply:
I will fix